tools/vsimporter/xib2nib/UIButton.cpp

```diff
@@ -253,22 +245,15 @@ void UIButton::InitFromStory(XIBObject* obj) {
     if (type) {
         if (strcmp(type, "roundedRect") == 0) {
             getAttrAndHandle("buttonType");
+             // TODO: Investigate why this value deviates from UIButtonType
```

Comment:
i really hate all of the magic constants in this codebase :'(

---
In reply to: [102358850](https://github.com/Microsoft/WinObjC/pull/2018#discussion_r102358850) [](ancestors = 102358850)